assets/javascripts/audit_log.js

```diff
@@ -8,10 +8,20 @@ function loadAuditLogTable ()
 {
     $('#audit_log_table').DataTable({
         lengthMenu: [20, 40, 100],
-        processing: true,
         serverSide: true,
         search: {search: searchquery},
-        ajax: {url: ajax_url, type: "GET", dataType: 'json'},
+        ajax: {
+            url: ajax_url,
+            type: 'GET',
+            dataType: 'json',
+            error: function(xhr) {
+                var msg = 'Search resulted in error ' + xhr.status;
+                if (xhr.responseJSON && xhr.responseJSON.error) {
```

Comment:
It makes sense to check these to avoid any further JavaScript errors during the error handling. So maybe it is worth checking whether `error` is actually an array (`… && Array.isArray(xhr.responseJSON.error)`).

Reply:
It should not be an array actually. I'm splitting the string if it contains line breaks.

I refactored the function into `handleAjaxQueryError` now which can be re-used.